<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1009</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1009</width>
    <height>676</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="home_page">
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="all_characters">
         <property name="text">
          <string>角色大全</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="character_query">
         <property name="text">
          <string>角色查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mine">
         <property name="text">
          <string>我的</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="Main_Widget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_0">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>60</y>
          <width>341</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>欢迎来到
原神角色图鉴~</string>
        </property>
       </widget>
       <widget class="QLabel" name="disply">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>561</width>
          <height>441</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">#scrollArea{background-color:transparent;

border-width:0;border-style:outset}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>973</width>
             <height>492</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="show_widget" name="show5" native="true"/>
               </item>
               <item row="0" column="0">
                <widget class="show_widget" name="show1" native="true"/>
               </item>
               <item row="0" column="2">
                <widget class="show_widget" name="show3" native="true"/>
               </item>
               <item row="2" column="1">
                <widget class="show_widget" name="show10" native="true">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="show_widget" name="show7" native="true"/>
               </item>
               <item row="1" column="1">
                <widget class="show_widget" name="show6" native="true"/>
               </item>
               <item row="0" column="1">
                <widget class="show_widget" name="show2" native="true"/>
               </item>
               <item row="1" column="3">
                <widget class="show_widget" name="show8" native="true"/>
               </item>
               <item row="0" column="3">
                <widget class="show_widget" name="show4" native="true"/>
               </item>
               <item row="2" column="0">
                <widget class="show_widget" name="show9" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="path_all">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>星级</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>性别</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>国家</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>使用武器</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>771</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="searchtext">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="preshow" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>211</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>查看</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>971</width>
             <height>490</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background color : rgba(255, 255, 255, 0)</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="show_widget" name="my1" native="true"/>
               </item>
               <item row="0" column="1">
                <widget class="show_widget" name="my2" native="true"/>
               </item>
               <item row="0" column="2">
                <widget class="show_widget" name="my3" native="true"/>
               </item>
               <item row="0" column="3">
                <widget class="show_widget" name="my4" native="true"/>
               </item>
               <item row="1" column="0">
                <widget class="show_widget" name="my5" native="true"/>
               </item>
               <item row="1" column="1">
                <widget class="show_widget" name="my6" native="true"/>
               </item>
               <item row="1" column="2">
                <widget class="show_widget" name="my7" native="true"/>
               </item>
               <item row="1" column="3">
                <widget class="show_widget" name="my8" native="true"/>
               </item>
               <item row="2" column="0">
                <widget class="show_widget" name="my9" native="true"/>
               </item>
               <item row="2" column="1">
                <widget class="show_widget" name="my10" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_6" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="path_my">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_my">
             <item>
              <property name="text">
               <string>星级</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>性别</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>国家</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>使用武器</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="import_mine"/>
   </widget>
   <widget class="QMenu" name="actionabout">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionadd"/>
    <addaction name="actiondelete"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="actionabout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="import_all">
   <property name="text">
    <string>导入全部</string>
   </property>
  </action>
  <action name="import_mine">
   <property name="text">
    <string>导入我的角色</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>添加角色</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>删除角色</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>show_widget</class>
   <extends>QWidget</extends>
   <header location="global">show_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
